content/CountdownWidget.kt

```diff
@@ -0,0 +1,22 @@
+fun secondsRemaining(now : Date, endDate : Date){
+  val nowMilliseconds = now.getTime()
+  val endMilliseconds = end.getTime()
+
+  val nowSeconds = nowMilliseconds / 1000
+  val endSeconds = endSeconds / 1000
+
+  return endSeconds - nowSeconds
+}
+
+
+@Test
+fun remaining_time_between_two_consecutive_days_is_86400_seconds(){
+  val oneDayInSeconds = 1*24*60*60
+  val format = SimpleDateFormat("yyyy-MM-dd")
+  val d1 = format.parse("2012-01-15")
+  val d2 = format.parse("2012-01-16")
```

Comment:
Using `SimpleDateFormat` to parse 2 days doesn't make sense.

Please use `Calendar` APIs to compute 2 consecutive days as follows:
```kotlin
val d1 = Date()
val cal = Calendar.getInstance();
cal.setTime(now);
cal.add(Calendar.DATE, 1);
val d2 = cal.getTime();
```